### DISCUSSION TITLE
Error and exit on Unifi LXC container

### DISCUSSION THREAD

**Discussion by Perjan-smul** [BassT23/Proxmox#161] [General]
When i run the update command my first container is my Unifi Container. When there is a update on het unifi controller package it will exit on a error. So i have to update the Unifi controller separately using a different update string. "apt-get update --allow-releaseinfo-change" as each time the unifi updates it changes the release info coherent with the version number. Then i have to rerun the update script to update the rest of my containers. 

Would it be possible to add the --allow-releaseinfo-change variable to the script?

**Comment by BassT23** [comment]
I also now had an unifi container. will try it out.
Hopefully this weekend I had time for it ;)

**Comment by BassT23** [comment]
how do you update the unifi controller?
I use a ubuntu lxc with controller installed via install script.
I had no error, but it also dont update the controller itself, only ubuntu :(

EDIT:
found it :) I use not the "stable" repo ;)

**Comment by BassT23** [comment]
fixed in latest beta ;)

**Comment by BassT23** [comment]
Fixed in latest release.
